public/app/frontend/src/components/breadcrumbs/breadcrumbs.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../../style';
+
+.breadcrumbs {
+    &__list {
+        display: flex;
+        align-items: center;
+        flex-wrap: wrap;
+    }
+
+    &__item {
```

Comment:
Could it do with a fraction of vertical spacing upon wrapping?

<img width="398" alt="image" src="https://github.com/user-attachments/assets/c81c8a5a-d8a4-42d7-b764-c6d2c5d0d404">